README.md

```diff
@@ -20,14 +20,22 @@ specific containers to use this project.
 ### Get started
 All your Drupal files should be placed into the folder "web".
 
+Add your env variable for dev.
+```sh
+$ cd to project root
+$ export BUILD_ENV=dev
```

Comment:
Hi @justclint,
I think I understand your issue now.
You are running the deployment script from your local, not from the container. This is why you have things like 'drush="docker-compose run --rm php drush"' in your drupal-update.sh file.
What I had in mind was more simple. If you are familar with Promet's current framework, we use the script inside vagrant. I would do the same with Docker, the script is executed inside the container, not your local. This is how you make accessing all the environment variables and other containers easy :)
You would start the script with: 
`docker-compose run --rm php ./../scripts/drupal-update.sh`
And that's it. The script should have access to everything it needs, because it's executed inside the container. Do you see any potential issues compared to what you were doing? Could you give it a try and update your script?
Let me know,
Thanks!
Luc